graphs/possible_bipartition.py

```diff
@@ -8,5 +8,25 @@ def possible_bipartition(dislikes):
         Time Complexity: ?
         Space Complexity: ?
     """
-    pass
+
+    if len(dislikes) == 0:
+        return False
+    first_item = list(dislikes.keys())[1]
+    stack = [first_item]
```

Comment:
It looks like you create a stack for a depth first search implementation, but you never use the stack. 

With depth first search, we want to loop through nodes while there is still a stack. For each node that we loop through (also called visiting), we want to look at each of its neighbors (which is like examining each of the current node's edges). If the neighbors have not yet been visited, we should add them to our stack.